pages/docs/cache.en-US.md

```diff
@@ -0,0 +1,94 @@
+# Custom Cache
+
+By default, SWR is using a global cache to store data sharing among all hooks. Besides that, there's a new way to customize your own cache provider.
+New property `cache` and a new API `createCache` are now introduced in `swr@beta`. This API is intended to resolve problem of using SWR with various, diverse storages.
+
+## Create Custom Cache
+
+### `createCache`
+
+This API receive a underlay cache `provider` as argument. Returns an object, with `cache` instance that could be consumed by SWR hooks,
+and `mutate` API to manipulate the corresponding cache. Note that it's not the global `mutate` API.
+
+```js
+const { mutate, cache } = createCache(provider)
+```
+
+You can pass down `cache` through SWRConfig or the `useSWR` hook options.
+
+```jsx
+import { SWRConfig, createCache } from 'swr'
+
+const provider = new Map()
+
+const { mutate, cache } = createCache(provider)
+
+// pass to SWR context
+<SWRConfig value={{ cache }}>
+  <Page />
+</SWRConfig>
+
+// or pass to hook options
+useSWR(key, fetcher, { cache })
+```
+
```

Comment:
(TODO: add note that mentions `createCache` should not be called inside render, it should be a global singleton.)